plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/controller/chat/messenger/ChatPromptHandler.kt

```diff
@@ -143,6 +145,24 @@ class ChatPromptHandler(private val telemetryHelper: TelemetryHelper) {
             }
     }
 
+    private fun extractCodeBlockLanguage(message: String): String {
```

Comment:
Can you please add a test for this?

Reply:
Moved `extractCodeBlockLanguage` to `shared/TextUtils` file and added test cases.